vector/src/main/java/im/vector/app/features/location/LocationSharingFragment.kt

```diff
@@ -176,6 +179,17 @@ class LocationSharingFragment @Inject constructor(
         views.mapView.zoomToLocation(event.userLocation.latitude, event.userLocation.longitude)
     }
 
+    private fun handleStartLiveLocationService(event: LocationSharingViewEvents.StartLiveLocationService) {
+        Intent(requireContext(), LocationSharingService::class.java)
+                .apply {
```

Comment:
as the `putX` methods on `Intent` are builder style, we could avoid the apply 

```java
Intent()
  .putExtra(foo, bar)
  .also { ContextCompat.startForegroundService(requireContext(), it) }
``` 

what do you think?

Reply:
Exactly, forgot that `puExtra` returns `Intent`. Done.